BotRoyaleUnity/Assets/Scripts/PartHealth.cs

```diff
@@ -20,6 +25,7 @@ void Start()
     // Update is called once per frame
     void Update()
     {
+        rb = transform.parent.gameObject.GetComponent<Rigidbody>();
```

Comment:
Not sure why this is here and not Awake(), is the parent's rigidbody going to be changing after the part is created?